dockerfiles/contracts-ci-linux/Dockerfile

```diff
@@ -49,8 +49,10 @@ RUN set -eux; \
 	cargo install cargo-dylint dylint-link && \
 	cargo install cargo-contract && \
 # download the latest `substrate-contracts-node` binary
-	curl -L "https://gitlab.parity.io/parity/substrate-contracts-node/-/jobs/artifacts/master/raw/artifacts/substrate-contracts-node-linux/substrate-contracts-node?job=build-linux" \
-		-o /usr/local/cargo/bin/substrate-contracts-node && \
+    curl -L -o substrate-contracts-node.zip 'https://gitlab.parity.io/parity/mirrors/substrate-contracts-node/-/jobs/1685841/artifacts/download?job=bud-linux' && \
```

Comment:
It's not a good choice to use artifacts from a job because we clean artifacts and job logs 1-2 times in a year.  If it's urgent we can merge it like this, if not I can fix it